JS/Clue.js

```diff
@@ -1,13 +1,52 @@
 class Clue {
+  constructor() {
+    this.question;
+    this.answerplace = 0; 
+    this.player = 0;
+    this.guessCount = 0;
+  }
+  checkAnswer(input) {
+    if( 'hello' != input && this.player === 3) {
+      this.player = 0;
+    } 
+    if ( round.currentClues[this.answerplace].answer !== input ) {
```

Comment:
What exactly does `answerplace` represent? An index of some sort? Is there a better variable name we can use here?